### PR TITLE
Adds initial CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by cacarrara**
Trying to start solve #23 this PR adds an initial Contributing.md file.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# How to contribute?
+
+##Steps to contribute
```
Comment (ManrajGrover) [inline CONTRIBUTING.md]: Needs spacing here

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+* Look for a issue or open a new one in [project
+  issues](https://github.com/ManrajGrover/halo/issues)
+* Fork the project
+* Clone to your machine based on your forked project
+* Create a new branch with an intuitive name. Take a look in concept of [feature
+  branch](https://martinfowler.com/bliki/FeatureBranch.html)
```
Comment (ManrajGrover) [inline CONTRIBUTING.md]: Needs to be in one line

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# How to contribute?
+
+##Steps to contribute
+
+* Look for a issue or open a new one in [project
+  issues](https://github.com/ManrajGrover/halo/issues)
```
Comment (ManrajGrover) [inline CONTRIBUTING.md]: Needs to be in one line

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  branch](https://martinfowler.com/bliki/FeatureBranch.html)
+* Code your change / fix / new feature
+* Run tests
+* When the tests pass you are free to commit and push
+* Open a Pull Request with a description and the issue reference
+
+## Best Practices
+
+Let's try to keep the code as simple and clean as we can. Some good pratices to
+follow during the contributing process:
```
Comment (ManrajGrover) [inline CONTRIBUTING.md]: One line please

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+- **Respect the PEP8**: don't forget to check the
+  [PEP8](https://www.python.org/dev/peps/pep-0008/) complains;
```
Comment (ManrajGrover) [inline CONTRIBUTING.md]: One line please

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+- **Write Tests**: **always** write tests for your code
+- **Keep the Cordiality**: be polite and kind; words like please and thank you
+  are welcome :)
```
Comment (ManrajGrover) [inline CONTRIBUTING.md]: One line please

**Comment by cacarrara** [comment]
I do not understand the purpose of move the Contributing file to `.github` directory. Seems that is a common practice keep this file in root.

**Comment by ManrajGrover** [comment]
@cacarrara It is a general convention and I feel it helps keeps source clear of unrelated files.

**Comment by ManrajGrover** [comment]
@cacarrara Just checking if you're still on this :)

**Pull request merged**